### PR TITLE
feat: update page hierachy

### PR DESCRIPTION
**Pull request by jeraldgan**
https://user-images.githubusercontent.com/16160955/187630660-6b188b3f-90a0-42ca-94f8-e5f5919b0aeb.mp4

## Changes
- Update page hierachy
- Update navbar

### CODE REVIEW

File: src/pages/demo/index.tsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+import { PageLink } from "@/modules/PageLink";
+
+export default function Home() {
+  const isFull = useBreakpointValue({ base: true, md: false });
+  return (
+    <div className="h-full">
+      <PageHead
+        append={false}
+        description="Home page description"
+        name="Home"
```
Comment (andrehadianto) [inline src/pages/demo/index.tsx:19]: is this intended?

**Review by andrehadianto — APPROVED** [review]
other than the comment, looks good to me!

File: src/pages/demo/index.tsx (hunk 1 above)
Comment (jeraldgan) [reply src/pages/demo/index.tsx:19]: ah right, thank you for the spot

Comment (jeraldgan) [reply src/pages/demo/index.tsx:19]: i realized it's merged, i will fix it in another PR LOL

**Pull request merged**